jasminewd2/index.d.ts

```diff
@@ -44,6 +44,13 @@ declare namespace jasmine {
     toThrowError(expected?: new (...args: any[]) => Error | Promise<new (...args: any[]) => Error>, message?: string | RegExp | Promise<string | RegExp>): Promise<void>;
   }
 
+  interface ArrayLikeMatchers<T extends ArrayLike<T>> extends Matchers<T> {
```

Comment:
The generic notation expects you to start with a name for the generic parameter
so you have to start with `<T ...`

The next part tells the compiles that the parameter `T` must be compatible with `ArrayLike<T>`

So basically it reads like:
> This interface is generic for a parameter named `T` which is of type `ArrayLike<T>`


Using just `ArrayLike<T>` would not give  the parameter any name and the compiler complains that he does not know `T`